waiter/src/waiter/kv.clj

```diff
@@ -69,13 +74,27 @@
 
 (defrecord LocalKeyValueStore [store]
   KeyValueStore
-  (retrieve [_ key _] (let [value (@store key)]
-                        (log/debug (str "(local) FETCH " key " => " (hashcode value)))
-                        value))
-  (store [_ key value] (do (log/debug (str "(local) STORE " key " => " (hashcode value)))
-                           (swap! store assoc key value)))
-  (delete [_ key] (do (log/debug (str "(local) DELETE " key))
-                      (swap! store dissoc key)))
+  (retrieve [_ key _]
+    (let [{:keys [value]} (@store key)]
```

Comment:
Is this a breaking change? When calling retrieve on previously stored values that weren't stored as `{:stats {}, :value}`.

Also separately, have we considered using zookeeper ctime and mtime of a znode?

Reply:
Disregard this, the zookeeper implementation uses the ctime and mtime, and we are okay with breaking changes to other kv-store implementations